src/api.rs

```diff
@@ -161,6 +170,13 @@ pub fn token_subset(
                         claims.prefixes.clone()
                     }
                 },
+                apps: {
+                    if let Some(ref apps) = args.apps {
+                        apps.clone()
+                    } else {
+                        claims.apps.clone()
+                    }
+                },
```

Comment:
While this pattern is the same as others in this block, I can't help but feel that something along the lines of:

```suggestion
                apps: args.apps.clone().unwrap_or_else(|| claims.apps.clone()),
```

might work more nicely.  Of course, I'm more used to options etc. and if you feel the expanded out approach is clearer for newcomers to the code, I'm happy to be ignored on this one.

Reply:
I'll leave it the same as the others for now, but maybe change it in the future.